CMakeLists.txt

```diff
@@ -9,10 +9,10 @@ project(
 find_package(Python COMPONENTS Interpreter Development REQUIRED)
 
 Python_add_library(_nvjitlinklib MODULE nvjitlink/_nvjitlinklib.cpp WITH_SOABI)
-
 find_package(
   CUDAToolkit 12.0 REQUIRED
 )
+
```

Comment:
Guessing these are left over from previous changes here. Maybe we can drop them unless we prefer them for some reason